test/session.ts

```diff
@@ -281,7 +297,7 @@ describe('Session', () => {
     it('should accept and pass gaxOptions to getMetadata', done => {
       const gaxOptions = {};
       session.request = config => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, gaxOptions);
```

Comment:
```suggestion
        assert.strictEqual(config.gaxOpts, gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `gaxOptions`.